homeassistant/components/sia/alarm_control_panel.py

```diff
@@ -44,40 +47,35 @@ class SIAAlarmControlPanelEntityDescription(
     """Describes SIA alarm control panel entity."""
 
 
-PREVIOUS_STATE = "previous_state"
-
-CODE_CONSEQUENCES: dict[str, StateType] = {
-    "PA": STATE_ALARM_TRIGGERED,
-    "JA": STATE_ALARM_TRIGGERED,
-    "TA": STATE_ALARM_TRIGGERED,
-    "BA": STATE_ALARM_TRIGGERED,
-    "CA": STATE_ALARM_ARMED_AWAY,
-    "CB": STATE_ALARM_ARMED_AWAY,
-    "CG": STATE_ALARM_ARMED_AWAY,
-    "CL": STATE_ALARM_ARMED_AWAY,
-    "CP": STATE_ALARM_ARMED_AWAY,
-    "CQ": STATE_ALARM_ARMED_AWAY,
-    "CS": STATE_ALARM_ARMED_AWAY,
-    "CF": STATE_ALARM_ARMED_CUSTOM_BYPASS,
-    "OA": STATE_ALARM_DISARMED,
-    "OB": STATE_ALARM_DISARMED,
-    "OG": STATE_ALARM_DISARMED,
-    "OP": STATE_ALARM_DISARMED,
-    "OQ": STATE_ALARM_DISARMED,
-    "OR": STATE_ALARM_DISARMED,
-    "OS": STATE_ALARM_DISARMED,
-    "NC": STATE_ALARM_ARMED_NIGHT,
-    "NL": STATE_ALARM_ARMED_NIGHT,
-    "BR": PREVIOUS_STATE,
-    "NP": PREVIOUS_STATE,
-    "NO": PREVIOUS_STATE,
-}
-
-entity_description_alarm = SIAAlarmControlPanelEntityDescription(
-    key="alarm_control_panel",
-    device_class="alarm",
-    code_consequences=CODE_CONSEQUENCES,
-    always_reset_availability=True,
+ENTITY_DESCRIPTION_ALARM = SIAAlarmControlPanelEntityDescription(
+    key=KEY_ALARM,
+    device_class=DEVICE_CLASS_ALARM,
```

Comment:
I'm not sure if device_class makes sense here.
That's a question for another reviewer.

Reply:
just relooked at that, I was mostly using it to unify the get_name, but now decided to get rid of get_name altogether in favor of using the same approach as for unique_id.